aspnetcore/includes/blazorwasm-preview-notice.md

```diff
@@ -1,4 +1,9 @@
 > [!IMPORTANT]
 > **Blazor WebAssembly in preview**
 >
-> *Blazor Server* is supported in ASP.NET Core 3.0. *Blazor WebAssembly* is in preview.
```

Comment:
How about we just update the current notice to say "*Blazor Server* is supported in ASP.NET Core 3.0. *Blazor WebAssembly* is in preview for ASP.NET Core 3.1"